### PR TITLE
docs: Remove XML warnings

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [x] Documentation content changes
 - [ ] Other, please describe:

## What is the current behavior?

We are having XML documentation warnings.

## What is the new behavior?

No warning.

## Checklist

Please check that your PR fulfills the following requirements:

- [x] Documentation has been added/updated.
- [ ] Automated tests for the changes have been added/updated.
- [ ] Updated [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) (if you introduced a breaking change).

### CODE REVIEW

File: src/ReviewService/ReviewService.cs [hunk 1]
```diff
@@ -20,6 +20,13 @@ public sealed class ReviewService<TReviewSettings> : IReviewService<TReviewSetti
 	private readonly IReviewSettingsSource<TReviewSettings> _reviewSettingsSource;
 	private readonly IList<IReviewCondition<TReviewSettings>> _reviewConditions;
 
+	/// <summary>
+	/// Initializes a new instance of the <see cref="ReviewService{TReviewSettings}"/> class.
+	/// </summary>
+	/// <param name="logger"><see cref="ILogger"/>.</param>
```
Comment (jeanplevesque) [inline src/ReviewService/ReviewService.cs]: These are probably wrong according the .Net guidelines. The text is supposed to be human readable sentences. This a good occasion to remind which parameter do what.

Comment (Soap-141) [reply src/ReviewService/ReviewService.cs]: Hmm were did found that? By doing some research I found some places that were saying this is good. Also I feel like it's actually readable by an human when Visual Studio or any IDE formats it. Unless you are only talking about the parameters.

Comment (jeanplevesque) [reply src/ReviewService/ReviewService.cs]: I referring to the parameters that just contain a class reference.

Comment (Soap-141) [reply src/ReviewService/ReviewService.cs]: Ok I can change this. I agree that it should be more than that.

Comment (Soap-141) [reply src/ReviewService/ReviewService.cs]: I updated them.

**Review by jeanplevesque — APPROVED** [review]

**Review by takla21 — APPROVED** [review]

**Pull request merged**